elasticsearch/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.podSecurityPolicy.create -}}
+{{- $fullName := include "uname" . -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ default $fullName .Values.podSecurityPolicy.name | quote }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ $fullName | quote }}
+spec:
```

Comment:
Good point. Done in [e7f76dc](https://github.com/elastic/helm-charts/pull/265/commits/e7f76dce3a69c451ff72830cbef57e62631663e1).

I've tried removing useless permissions on the default values - it does work on my k8s setup but it might be reduced further or causing issues on other setup?

Reply:
It's disabled by default so I'm not too worried about making sure it works for everyone. Especially now that it is configurable 👍